incubator/dcos_spark_install/preparation/script.md

```diff
@@ -0,0 +1,55 @@
+# Creating a Cluster
+
+We will use the Azure CLI 2.0 to quickly create an Azure Container
+Services cluster. Ensure you have the Azure CLI installed and have
+logged in using the `az login` command.
+
+We first need to set a few environment variables we will use later:
+
+```
+RESOURCE_GROUP=acs-spark-demo
+SERVICE_NAME=acs-dcos-spark
+DNS_NAME_PREFIX=acs-dcos-spark
+REGION=eastus
+```
+
+```
+echo region is $REGION
+```
```

Comment:
NOTE: this does not work. Environment variables are not carried across from one command execution to the next. It is possible to pass in environment variables to SimDem execution environments but the format is JSON. We need to think about how to make these environment variables work.